webdriver/newsession/response.py

```diff
@@ -0,0 +1,54 @@
+# META: timeout=long
+
+import uuid
+
+def test_resp_sessionid(new_session):
+    resp, _ = new_session({"capabilities": {}})
+    assert isinstance(resp["sessionId"], unicode)
+    uuid.UUID(hex=resp["sessionId"])
+
+
+def test_resp_capabilites(new_session):
+    resp, _ = new_session({"capabilities": {}})
+    assert isinstance(resp["sessionId"], unicode)
+    assert isinstance(resp["capabilities"], dict)
+    assert {"browserName",
+            "browserVersion",
+            "platformName",
+            "acceptInsecureCerts",
+            "setWindowRect",
+            "timeouts",
+            "proxy",
+            "pageLoadStrategy"}.issubset(
+                set(resp["capabilities"].keys()))
+
+
+def test_resp_data(new_session, platform_name):
+    resp, _ = new_session({"capabilities": {}})
+
+    assert isinstance(resp["capabilities"]["browserName"], unicode)
+    assert isinstance(resp["capabilities"]["browserVersion"], unicode)
+    if platform_name:
+        assert resp["capabilities"]["platformName"] == platform_name
+    assert resp["capabilities"]["acceptInsecureCerts"] is False
+    assert isinstance(resp["capabilities"]["setWindowRect"], bool)
+    assert resp["capabilities"]["timeouts"] == {
```

Comment:
Yes there is. The spec literally defines that you return those three properties.